srcpkgs/grub-btrfs/files/grub-btrfs/run

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+[ -r conf ] && . ./conf
+exec 2>&1
```

Comment:
You probably want to put this above the conf source just in case there are some error messages printed while sourcing the conf file.